### PR TITLE
Editor update and misc compiler warning fixes.

### PR DESCRIPTION
**Pull request by StephenHodgson**
Updated Editor to 2017.2.0b11.
Updated obsolete quality settings usages.
fixed misc warnings.

### CODE REVIEW

File: ProjectSettings/ProjectVersion.txt [hunk 1]
```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.2.0b10
```
Comment (keveleigh) [inline ProjectSettings/ProjectVersion.txt]: Please update the readme as well

File: Assets/HoloToolkit/Input/Scripts/Utilities/MixedRealityCameraManager.cs [hunk 2]
```diff
@@ -13,19 +13,27 @@ public class MixedRealityCameraManager : Singleton<MixedRealityCameraManager>
     {
         [Tooltip("The near clipping plane distance for an opaque display.")]
         public float NearClipPlane_OpaqueDisplay = 0.3f;
+
         [Tooltip("Values for Camera.clearFlags, determining what to clear when rendering a Camera for an opaque display.")]
         public CameraClearFlags CameraClearFlags_OpaqueDisplay = CameraClearFlags.Skybox;
+
+        [Tooltip("Background color for a transparent display.")]
+        public Color BackgroundColor_OpaqueDisplay = Color.black;
```
Comment (keveleigh) [inline Assets/HoloToolkit/Input/Scripts/Utilities/MixedRealityCameraManager.cs]: I don't see this showing up in the inspector:

image

File: ProjectSettings/ProjectVersion.txt (hunk 1 above)
Comment (keveleigh) [inline ProjectSettings/ProjectVersion.txt]: Please update the readme as well.

File: Assets/HoloToolkit/Input/Scripts/Editor/MixedRealityCameraMangerEditor.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
+﻿using HoloToolkit.Unity.InputModule;
```
Comment (keveleigh) [inline Assets/HoloToolkit/Input/Scripts/Editor/MixedRealityCameraMangerEditor.cs]: Please add the header!

File: Assets/HoloToolkit/Input/Scripts/Utilities/MixedRealityCameraManager.cs [hunk 4]
```diff
@@ -69,6 +77,7 @@ public void ApplySettingsForOpaqueDisplay()
             Debug.Log("Display is Opaque");
             Camera.main.clearFlags = CameraClearFlags_OpaqueDisplay;
             Camera.main.nearClipPlane = NearClipPlane_OpaqueDisplay;
+            Camera.main.backgroundColor = BackgroundColor_OpaqueDisplay;
```
Comment (keveleigh) [inline Assets/HoloToolkit/Input/Scripts/Utilities/MixedRealityCameraManager.cs]: If solid color isn't set for the clear flags, does changing this value do anything that could be unwanted?

Comment (StephenHodgson) [reply Assets/HoloToolkit/Input/Scripts/Utilities/MixedRealityCameraManager.cs]: no.

File: ProjectSettings/ProjectVersion.txt (hunk 1 above)
Comment (StephenHodgson) [reply ProjectSettings/ProjectVersion.txt]: Done

**Pull request merged**